spec/models/course/assessment_spec.rb

```diff
@@ -57,6 +57,34 @@
           it { is_expected.to be_valid }
         end
       end
+
+      context 'when the assessment is set to be autograded' do
+        subject { assessment }
+
+        context 'when the assessment has a non-autograded question' do
+          let!(:non_autograded_question) do
+            create(:course_assessment_question, assessment: assessment)
+          end
+
+          it 'is not valid' do
+            assessment.autograded = true
```

Comment:
Tried that, but what happens is that I then cannot create the `:non_autograded_question`. So even if I move this up into a `before` block, I would have to build a `:non_autograded_question` in the `it` block - so it becomes a lesser of 2 evils. 

Not sure how else I can do it though. Unless I just build everything?